interface/main/main_screen.php

```diff
@@ -116,6 +116,10 @@
 if ($GLOBALS['new_tabs_layout']) {
   $_SESSION['frame1url'] = $frame1url;
   $_SESSION['frame1target'] = $frame1target;
+  // mdsupport - Apps processing invoked for valid app selections from list
+  if ((isset($_POST['appChoice'])) && ($_POST['appChoice'] !== '*OpenEMR')) {
```

Comment:
actually, I would also incorporate this same conditional in line 116 above. Thus, it will open the app in app mode no matter what the new_tabs_layout setting is.